File: back/KaemeWebApi/Dominio/Models/Peca.cs
namespace Dominio.Models
{
    public partial class Peca
    {
        public int? Id { get; set; }
        public string Codigo { get; set; }
        public string Descricao { get; set; }
        public decimal Valor_Compra { get; set; }
        public decimal Valor_Venda { get; set; }
        public int Quantidade { get; set; }
        public bool Ativo { get; set; }
        public int Tipo_Peca_Id { get; set; }
        public int Fornecedor_Id { get; set; }
        public string? Observacao { get; set; }
        public DateTime Data_Criacao { get; set; }
        public int Criado_Por { get; set; }

    }
}

File: front/kaemeweb/src/app/modules/shared/models/peca.ts
export interface Peca {
  id: number | null;
  codigo: string;
  descricao: string;
  valor_Compra: number;
  valor_Venda: number;
  quantidade: number;
  ativo: boolean;
  tipo_Peca_Id: number;
  fornecedor_Id: number;
  observacao?: string | null;
  data_Criacao: Date;
  criado_Por: number;

  // Propriedades de visualização
  tipo_Peca_Descricao?: string | null;
  fornecedor?: string | null;

}
